app/javascript/main.jsx

```diff
@@ -31,31 +32,33 @@ const queryClient = new QueryClient();
 const root = (
   <QueryClientProvider client={queryClient}>
     <AuthProvider>
-      <Router>
-        <Routes>
-          <Route path="/" element={<App />}>
-            <Route index element={<HomePage />} />
-            <Route path="/signup" element={<Signup />} />
-            <Route path="/signin" element={<SignIn />} />
-            <Route path="/forget_password" element={<ForgetPassword />} />
-            <Route path="/reset_password/:token" element={<ResetPassword />} />
-            <Route path="/profile" element={<Profile />} />
-            <Route path="/admin" element={<Navigate to="/admin/users" replace />} />
-            <Route path="/admin/users" element={<ManageUsers />} />
-            <Route path="/admin/edit_user/:userId" element={<EditUser />} />
-            <Route path="/admin/server-recordings" element={<ServerRecordings />} />
-            <Route path="/admin/server-rooms" element={<ServerRooms />} />
-            <Route path="/admin/room-configuration" element={<RoomConfig />} />
-            <Route path="/admin/site-settings" element={<SiteSettings />} />
-            <Route path="/admin/roles" element={<Roles />} />
-            <Route path="/admin/roles/edit/:roleId" element={<EditRole />} />
-            <Route path="/rooms" element={<Rooms />} />
-            <Route path="/rooms/:friendlyId" element={<Room />} />
-            <Route path="/rooms/:friendlyId/join" element={<RoomJoin />} />
-            <Route path="*" element={<h1 className="text-center">404</h1>} />
-          </Route>
-        </Routes>
-      </Router>
+      <React.Suspense fallback="Loading...">
```

Comment:
What's this change for?

Reply:
React Suspense is to be used with React "lazy-loading", which is not something we are using right now. 
Removed React Suspense from the Router tree.